interactive/models.py

```diff
@@ -71,6 +71,13 @@ class User(AbstractBaseUser, PermissionsMixin):
         unique=True,
         error_messages={"unique": "A user with this email address already exists."},
     )
+    organisation = models.CharField(
+        max_length=100, verbose_name="Organisation", default=None, blank=True, null=True
+    )
+    job_title = models.CharField(
+        max_length=100, verbose_name="Job title", default=None, blank=True, null=True
+    )
```

Comment:
We shouldn't make textual fields nullable, it gives us two "empty" states, an empty string and null.  You can make them non-nullable and set a default of an empty string to avoid it.

You might want to also consider using TextFields here so you don't have to set a max_length, for this type of data they're always arbitary.  Postgres (and SQLite) implement them with the same underlying data type so there's no cost to using them.  The only pain in Django is having to set the widget type in the admin.

Reply:
I've found working with TextFields by default much easier over the years.  The max_length is typically arbitrary so you end up updating it only when you hit the limit during a legitimate use.  The only downside is having to tell Django to use `TextInput` widgets on autogenerated forms (eg in the Admin).